### DISCUSSION TITLE
Should we do something more professional for supporting the "community" building aspects then GitHub? Like an actual "Community Platform" service?

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/riverscapes-website-jekyll#53] [Ideas]
With the RC web presence overhaul, we make a big deal about supporting and growing communities. We DO NOT want to reinvent social media wheels or solutions that are already out there. However, are we better @philipbaileynar and @MattReimer to invest in a service that allows us to bake this into our riverscapes.net  site and experience? 

I'm thinking something like [hiverbrite](https://hivebrite.com/professional-communities?utm_campaign=google_longtailkeywords__na_N_A__search_cpc&utm_medium=cpc&utm_source=google&utm_term=%2Bcommunity+%2Bplatforms+%2Bsoftware) or one these [Community Platform services](https://www.mightynetworks.com/resources/community-platforms). Thoughts?

**Comment by MattReimer** [comment]
I'm a bit out of my wheelhouse with this stuff so I'm not really sure what the needs are or what options support those best in general:

- simpler the better
- out of the box solutions are best
- keep maintenance as low as possible 

Happy to talk about this but it's not a subject I'm confident enough to have strong opinions about

**Reply by joewheaton** [reply]
I'm at the point where I would rather we pay for a professional service, then stand something up ourselves and have one more thing we have to maintain. I look forward to chatting with you about this.

**Comment by joewheaton** [comment]
@dominique-shore can you please give this a similar treatment to what you did for HelpDesk services. FYI @philipbaileynar

**Reply by dominique-shore** [reply]
I'd be interested to hear more about your goals for this platform, as I'm not too familiar with Community Platforms, but happy to look into it. 

I will say - the FreshDesk knowledge base does have a forum that customers could use to interact with our devo team, and other users. It would be included in the Customer Support software, so one less subscription service, but I'm not sure that is covering all the community-building services you are looking for?
